### ISSUE TITLE
Notes on generating reports in PDF format

### ISSUE THREAD

**Issue by sebastfr** [eprints/irstats2#53]
the free tool wkhtmltopdf works nicely nowadays and it shouldn't be too hard to be able to export a report page (eg demoprints.eprints.org/cgi/stats/report) to a PDF which can then be emailed to admins.

installable via apt-get for the ubunters

sample: wkhtmltopdf <url> <output.pdf>

**Comment by drn05r** [comment]
I don't think this is a relevant feature as I have not had anyone desperately requesting this feature in the last 10 years.
